lms/static/scripts/frontend_apps/services/client-rpc.js

```diff
@@ -24,6 +24,43 @@ import { JWT } from '../utils/jwt';
  * @typedef {import('../config').ClientConfig} ClientConfig
  */
 
+/**
+ * Content provider logo details.
+ *
+ * @typedef ContentInfoLogo
+ * @prop {string} logo
+ * @prop {string} title
+ * @prop {string} link
+ */
+
+/**
+ * Metadata for the current document, for display in the content info banner.
+ *
+ * @typedef ContentInfoItem
+ * @prop {string} title - Title of the current article, chapter etc.
+ * @prop {string} containerTitle - Title of the journal issue, book etc. which
+ *   contains the current work
+ */
+
+/**
+ * Links related to the current document, for display in the content info banner.
+ *
+ * @typedef ContentInfoLinks
+ * @prop {string} [previousItem] - Previous item in the book, journal etc.
+ * @prop {string} [nextItem] - Next item in the book, journal etc.
+ */
```

Comment:
Remind me: You'll be adding a link to the current item in the...`ContentInfoLinks`?

Reply:
Yes. The plan is to add a `currentItem` link here.